lib/class-wp-theme-json-resolver.php

```diff
@@ -151,6 +151,34 @@ public static function get_presets_to_translate() {
 		return $theme_json_i18n;
 	}
 
+	/**
+	 * Translates a chunk of the loaded theme.json structure.
+	 *
+	 * @param array  $array_to_translate The chunk of theme.json to translate.
+	 * @param string $key                The key of the field that contains the string to translate.
+	 * @param string $context            The context to apply in the translation call.
+	 * @param string $domain             Text domain. Unique identifier for retrieving translated strings.
+	 *
+	 * @return array Returns the modified $theme_json chunk.
+	 */
+	private static function translate_theme_json_chunk( $array_to_translate, $key, $context, $domain ) {
+		if ( null === $array_to_translate ) {
+			return $array_to_translate;
+		}
+
+		foreach ( $array_to_translate as $item_key => $item_to_translate ) {
+			if ( empty( $item_to_translate[ $key ] ) ) {
+				continue;
+			}
+
+			// phpcs:ignore WordPress.WP.I18n.LowLevelTranslationFunction,WordPress.WP.I18n.NonSingularStringLiteralText,WordPress.WP.I18n.NonSingularStringLiteralContext,WordPress.WP.I18n.NonSingularStringLiteralDomain
+			$array_to_translate[ $item_key ][ $key ] = translate_with_gettext_context( $array_to_translate[ $item_key ][ $key ], $context, $domain );
+			// phpcs:enable
```

Comment:
You don't need `phpcs:enable` here. You would only need it to re-enable after a `phpcs:disable`. But using that combo is rarely needed.

So only the `phpcs:ignore` above should be enough

Reply:
I copied it from another place, so I will update all instances. Thank you for raising it 👍🏻